opt/cs50/bin/http-server

```diff
@@ -9,6 +9,18 @@ port="-p 8080"
 options="--no-dotfiles"
 t="-t0"
 
+# Formatting
+bold=$(tput bold)
+normal=$(tput sgr0)
+
+# Check for app.py or wsgi.py
```

Comment:
I added detection for `wsgi.py` here just so we match flask's default application discovery behavior:
https://flask.palletsprojects.com/en/3.0.x/quickstart/

Reply:
Ah nice.